modules/shared/components/molecules/UploadingImage/UploadingImage.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import Image from 'next/image';
+import type { IUploadingImage } from './IUploadingImage';
+import styles from './UploadingImage.module.css';
+import TextInput from '../../atoms/TextInputs/TextInput';
+import * as ETextInput from '../../atoms/TextInputs/types/ETextInput';
+import VerticalThreeDots from '../../icons/verticalThreeDots.svg';
+
+const UploadingImage: FC<IUploadingImage.IProps> = ({
+  imageUrl = 'https://source.unsplash.com/random',
```

Comment:
I don't think it's a good idea to have a default imageUrl, maybe instead the component should render if it's not passed.